web/src/app/layout.tsx

```diff
@@ -68,7 +67,32 @@ export default function RootLayout({
             className="bg-cover bg-center mb-6 p-6 pb-0"
             style={{ backgroundImage: `url(${bgImage.src})` }}
           >
-            <h1 className="text-3xl mb-8 pixel-text">Quadradius Guide</h1>
+            <div className="flex flex-row justify-between">
+              <h1 className="text-3xl mb-8 pixel-text inline">
+                Quadradius Guide
+              </h1>
+              <div className="justify-end mb-4 inline-block">
+                <a
+                  href="https://discord.com/channels/1197227968112627802"
```

Comment:
This should be taken from [config.json](https://github.com/Fruktus/QuadradiusPreservationProject/pull/167/files#diff-a9f42349b651bd9f34053283ea489e364088196b300333da9a977c6115da76a4) especially since this is a link to channel and not an invite